Libplanet.Tests/Net/SwarmTest.Preload.cs

```diff
@@ -830,297 +591,6 @@ async Task MineBlocks()
-                }
-
-                for (int i = 1; i <= repeat; i++)
-                {
-                    if (i == 3 || i == 7 || i == 11 || i == repeat)
-                    {
-                        var state = chain1.GetState(
-                            swarm0.Address,
-                            blockHashes[i],
-                            stateCompleter: StateCompleters<DumbAction>.Reject
-                        );
-                        Assert.NotNull(state);
-                        Assert.Equal((Integer)i, (Integer)state);
-                    }
-                    else
-                    {
-                        Assert.Throws<IncompleteBlockStatesException>(() =>
-                        {
-                            chain1.GetState(
-                                swarm0.Address,
-                                blockHashes[i],
-                                stateCompleter: StateCompleters<DumbAction>.Reject
-                            );
-                        });
-                    }
-                }
-
-                await swarm2.AddPeersAsync(new[] { swarm1.AsPeer }, null);
-                await swarm2.PreloadAsync(
-                    trustedStateValidators: new[] { swarm1.Address }.ToImmutableHashSet());
-
-                Assert.Equal(chain1.BlockHashes, chain2.BlockHashes);
-
-                foreach (BlockChain<DumbAction> chain in new[] { chain1, chain2 })
-                {
-                    IValue blockActionState = chain.GetState(
-                        swarm0.Address,
-                        stateCompleter: StateCompleters<DumbAction>.Reject
-                    );
-                    Assert.NotNull(blockActionState);
-                    Assert.Equal((Integer)repeat, (Integer)blockActionState);
-                }
-
-                for (int i = 1; i <= repeat; i++)
-                {
-                    if (i == 11 || i == repeat)
-                    {
-                        var state = chain2.GetState(
-                            swarm0.Address,
-                            blockHashes[i],
-                            StateCompleters<DumbAction>.Reject
-                        );
-                        Assert.NotNull(state);
-                        Assert.Equal((Integer)i, (Integer)state);
-                    }
-                    else
-                    {
-                        Assert.Throws<IncompleteBlockStatesException>(() =>
-                        {
-                            chain2.GetState(
-                                swarm0.Address,
-                                blockHashes[i],
-                                StateCompleters<DumbAction>.Reject
-                            );
-                        });
-                    }
-                }
-            }
-            finally
-            {
-                await StopAsync(swarm0);
-                await StopAsync(swarm1);
-            }
-        }
-
```

Comment:
I removed this test because `Swarm<T>.FindNextStatesChunkSize` is removed. `FindNextStatesChunkSize` was a variable for `Swarm<T>.TransferRecentStates()` removed.